### ISSUE TITLE
dash '-' instead of dot '.'

### ISSUE THREAD

**Issue by SetiZ** [mistakster/postcss-pipeline-webpack-plugin#57]
Is it possible to choose not to put a dot and put a dat xxx-critical.css.
Also I use mini-css-extract-plugin, 
`new MiniCssExtractPlugin({
      filename: 'styles-[hash:8].css'
    }),`

Is there a was to make the `-critical` before the hash? (it's kinda like #41 )

**Comment by SetiZ** [comment]
closed by #58

**Comment by mistakster** [comment]
Hi SetiZ!

I’ve just published version 5.1.0 which let you configure destination filenames as you wish.

I added `transformName` option. You can provide a function which returns the new name for each file. For example

```
plugins: [
  new MiniCssExtractPlugin({ filename: 'styles-[hash:8].css' }),
  new PostCssPipelineWebpackPlugin({
    processor: postcss([criticalSplit()]),
    transformName: name => 'critical-' + name
  })
]
```

And you get `styles-12345678.css` and `critical-styles-12345678.css` in this case.

Actually, you can do a complex find/replace operation with RegExp inside the function.

By the way, `prefix` and `suffix` options remain the same if `transformName` is not defined.

Hope this helps you. And thank you for your PR.

Let me know if you have any questions.


Cheers!
